src/wechaty_plugin_contrib/contrib/chat_history_plugin/plugin.py

```diff
@@ -36,46 +60,38 @@ def __init__(self, options: Optional[ChatHistoryPluginOptions] = None):
             self.chat_history_path = os.path.join(os.getcwd(), 'chathistory')
         if not os.path.exists(self.chat_history_path):
             os.makedirs(self.chat_history_path)
-        self.database_path = os.path.join(
-            self.chat_history_path, 'chathistory.db')
+        if not options.chat_history_database:
+            self.chat_history_database = 'sqlite+aiosqlite:///chathistory.db'
+        else:
+            self.chat_history_database = options.chat_history_database
```

Comment:
Wo hoooo ~ That's what I would do: 

```python
self.chat_history_database = options.chat_history_database or 'sqlite+aiosqlite:///chathistory.db'
```
This code is much more streamlined and readable. How do you think about it ?

Reply:
The line will be too long and violate PEP8/Pylint maximum line length of characters, which may cause CI failure~

Reply:
So ,you can have it two lines to pass the lint test.